src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <>
+      {movies.map(movie => (
+        <MovieCard
+          title={movie.title}
+          description={movie.description}
+          imgUrl={movie.imgUrl}
+          imdbUrl={movie.imdbUrl}
```

Comment:
add key prop. it should be some unique value, may be in this case it can be movie.imdbUrl

Reply:
done ) it is an imdbId)

Reply:
there should be key property
```suggestion
          imdbUrl={movie.imdbUrl}
          key={movie.imdbId}
```